src/framework/shortcuts/qml/MuseScore/Shortcuts/EditShortcutDialog.qml

```diff
@@ -69,13 +71,36 @@ Dialog {
 
             spacing: 20
 
+            RowLayout {
+                anchors.horizontalCenter: parent.horizontalCenter
+                spacing: 8
+
+                StyledIconLabel {
+                    horizontalAlignment: Text.AlignHCenter
+
+                    iconCode: root.currentShortcut["icon"]
+                    font: ui.theme.iconsFont
+
+                    visible: !isEmpty
+                }
+
+                StyledTextLabel {
+                    horizontalAlignment: Text.AlignHCenter
+
+                    text: root.currentShortcut["title"]
+                    font: ui.theme.bodyBoldFont
+
+                    visible: !isEmpty
+                }
+            }
+
             StyledTextLabel {
                 anchors.horizontalCenter: parent.horizontalCenter
 
                 width: parent.width
-                text: qsTrc("shortcuts", "Define keyboard shortcut")
-                horizontalAlignment: Text.AlignLeft
-                font:ui.theme.headerBoldFont
+                text: qsTrc("shortcuts", "Press a key or sequence of keys to assign a shortcut to this function.")
+                horizontalAlignment: Text.AlignHCenter
+                font: ui.theme.bodyFont
```

Comment:
Since these two properties have their default value, you may as well remove them.